examples/simple-todos/simple-todos.js

```diff
@@ -19,13 +19,13 @@ if (Meteor.isClient) {
 
   Template.body.helpers({
     tasks: function () {
+      var options = {};
+
       if (Session.get("hideCompleted")) {
-        // If hide completed is checked, filter tasks
-        return Tasks.find({checked: {$ne: true}}, {sort: {createdAt: -1}});
-      } else {
-        // Otherwise, return all of the tasks
-        return Tasks.find({}, {sort: {createdAt: -1}});
+        options = {checked: {$ne: true}};
```

Comment:
I'm not sure if we should take this, but if we do we should not call this variable options: options is the name of the second argument to `find`, not the first.